backend/inspirehep/records/cli.py

```diff
@@ -338,3 +341,84 @@ def remove_bai_from_literature_records(
     for chunk in chunker(documents, 100):
         uuids = [record.meta.id for record in chunk]
         remove_bai_from_literature_authors.delay(uuids)
+
+
+@click.group()
+def legacy_records():
+    """Commands for LegacyRecordsMirror"""
+
+
+@legacy_records.command(help="Export marcxml column from LegacyRecordsMirror table")
+@click.option(
+    "--dir-path",
+    help="Path where the xml files will be saved.",
+    default="/tmp/legacy_xmls",
+    show_default=True,
+)
+@with_appcontext
+def export_xml(dir_path):
+    if not os.path.exists(dir_path):
+        try:
+            os.mkdir(dir_path)
+        except Exception as e:
+            LOGGER.info(f"There was a problem creating the dir: {e}")
+    if os.path.exists(dir_path):
+        try:
+            query = LegacyRecordsMirror.query.all()
+            for row in query:
+                file_path = os.path.join(dir_path, f"{row.recid}.xml")
+                xml = open(file_path, "wb")
+                xml.write(row.marcxml)
+                xml.close()
+        except Exception as e:
+            LOGGER.info(f"There was a problem exporting the column: {e}")
+
+    LOGGER.info(
+        f"{len([f for f in os.listdir(dir_path) if os.path.isfile(os.path.join(dir_path, f))])} marcxml rows exported to xml."
```

Comment:
We use structlog, so you wirte a message like LOGGER.info("marcxml rows exported to xml.", rows_number=rows_number"). Can you also extract this list comprehension to a variable and make it more readable?